app/cells/decidim/alternative_landing/content_blocks/base_cell.rb

```diff
@@ -18,14 +18,22 @@ def participatory_spaces
           ].flatten.compact
         end
 
-        def available_components(manifest_name)
-          Decidim::Component.published.where(participatory_space: participatory_spaces, manifest_name: manifest_name).map do |component|
+        def available_components
+          components.where(manifest_name: manifest_name).map do |component|
```

Comment:
```suggestion
          @available_components ||= components.where(manifest_name: manifest_name).map do |component|
```
maybe memoize here this mapping

Reply:
Great :D